tide/master/control/DisplayGroupController.cpp

```diff
@@ -48,10 +48,9 @@
 
 namespace
 {
-bool _exceeds(const auto& size, const auto& other)
-{
+static auto _exceeds = [](const auto& size, const auto& other) {
     return size.width() > other.width() && size.height() > other.height();
-}
+};
```

Comment:
?

Reply:
The explanation is in the commit message for this change, auto parameters for non-lambda functions is not in C++14 standard so clang refuses to compile it (unlike gcc). The solution is either this (a static lambda) or declare a standard template function (with typename XYZ).

Reply:
true